hudi-utilities/pom.xml

```diff
@@ -131,6 +131,12 @@
       <artifactId>log4j</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Remove this please. We are using log4j directly. This causes issues with shading

Reply:
Right now SLF4j and log4j both are used in Hudi (realized this during the test suite logging). We use it in just the HoodieClientTestHarness class. Follow up task to remove 1 of them is here -> https://issues.apache.org/jira/browse/HUDI-1026